bitnami/mxnet/templates/scheduler-deployment.yaml

```diff
@@ -0,0 +1,160 @@
+{{- if eq .Values.mode "distributed" }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "mxnet.fullname" . }}-scheduler
+  labels:
+    app.kubernetes.io/name: {{ include "mxnet.name" . }}
+    helm.sh/chart: {{ include "mxnet.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/component: "scheduler"
+spec:
+  replicas: 1 
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "mxnet.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+      app.kubernetes.io/component: "scheduler"
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "mxnet.name" . }}
+        helm.sh/chart: {{ include "mxnet.chart" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        app.kubernetes.io/managed-by: {{ .Release.Service }}
+        app.kubernetes.io/component: "scheduler"
+    spec:
+      {{- include "mxnet.imagePullSecrets" . | nindent 6 }}
+      {{- if .Values.securityContext.enabled }}
+      securityContext:
+        fsGroup: {{ .Values.securityContext.fsGroup }}
+        runAsUser: {{ .Values.securityContext.runAsUser }}
+      {{- end }}
+      {{- if .Values.nodeSelector }}
+      nodeSelector: {{ toYaml .Values.nodeSelector | nindent 8 }}
+      {{- end }}
+      {{- if .Values.tolerations }}
+      tolerations: {{ toYaml .Values.tolerations | nindent 8 }}
+      {{- end }}
+      {{- if .Values.affinity }}
+      affinity: {{ toYaml .Values.affinity | nindent 8 }}
+      {{- end }}
+      {{- if .Values.cloneFilesFromGit.enabled }}
+      initContainers:
+      - name: git-clone-repository
+        image: {{ include "git.image" . }}
+        imagePullPolicy: {{ .Values.git.pullPolicy | quote }}
+        command:
+        - /bin/sh
+        - -c
+        - |
+          git clone {{ .Values.cloneFilesFromGit.repository }} /app
+          cd /app
+          git checkout {{ .Values.cloneFilesFromGit.revision }}
+        volumeMounts:
+        - name: git-cloned-files
+          mountPath: /app
+      {{- end }}
+{{- if .Values.initContainers }}
```

Comment:
can you indent this code for readability? and also use `nindent` in the following line?

Reply:
Done